prow/jobs/kyma/tests/monitoring/monitoring.yaml

```diff
@@ -16,7 +16,7 @@ job_template: &job_template
         command:
           - "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/build.sh"
         args:
-          - "/home/prow/go/src/github.com/kyma-project/kyma/tests/test-logging-monitoring"
+          - "/home/prow/go/src/github.com/kyma-project/kyma/tests/monitoring"
```

Comment:
Same `job_template` is being used for master and release jobs. If you changed this, release jobs will also be affected. Do you really want this? Are you renaming the component in release branches as well? If not, please create another job template with the change and use it for master jobs.

Reply:
Refactored as needed